### PR TITLE
refactor: add quotes around node-wot results and add comment column

### PR DESCRIPTION
**Pull request by danielpeintner**
replacement for https://github.com/w3c/wot-testing/pull/485 by adding quotes

### CODE REVIEW

File: data/input_2022/TD/node-wot/node-wot.csv [hunk 1]
```diff
@@ -1,109 +1,109 @@
...
-tm-tmRef-overwrite-process,null,The process to overwrite MUST follow the JSON Merge Patch algorithm as defined in [RFC7396] where the content of the referenced definition is patched with the new provided JSON name-value pairs.
-tm-tmRef-overwrite-semantic-meaning,null,Similar to tm:extends and to keep the semantic meaning, definitions SHOULD NOT be overwritten in such a way that possible instance values are no longer valid compared to the origin referenced definition.
-tm-tmRef-usecase,pass,For importing pieces of definitions of one or more existing Thing Models, the tm:ref term is introduced that provides the location of an existing (sub-)definition that SHOULD be reused.
-tm-tmRef2,null,Every time tm:ref is used, the referenced pre-definition and its dependencies (e.g., by context extension) MUST be assumed at the new defined definition.
-tm-tmRequired-resolver,TODO,The JSON Pointers of tm:required MUST resolve to an entire interaction affordance Map definition.
-tm-versioning,null,When the Thing Model definitions change over time, this SHOULD be reflected in the version container.
+"ID", "Status", "Comment", "Description"
```
Comment (egekorkan) [inline data/input_2022/TD/node-wot/node-wot.csv:1]: Please use Assertion instead of Description

**Review by egekorkan — CHANGES_REQUESTED** [review]
Also please have a look at https://github.com/w3c/wot-testing/blob/d21517ddee7e5ae86b43a816bc50520c1772df08/events/2022.12.Online/TD/manual-csv-changelogs/node-wot.md and update your results :)

File: data/input_2022/TD/node-wot/node-wot.csv (hunk 1 above)
Comment (mmccool) [inline data/input_2022/TD/node-wot/node-wot.csv:1]:
```suggestion
"ID", "Status", "Comment", "Assertion"
```

**Comment by mmccool** [comment]
I'm going to merge this so I can use the new results, but I noticed that it looks like only the first line of the assertion is being picked up, see for example thing-model-td-generation-processor-extends.  In my tooling (and in the new template.csv files I have been generating) I was careful to pick up the whole assertion (and turn newlines into whitespace), but it looks like that was not done here.

File: data/input_2022/TD/node-wot/node-wot.csv (hunk 1 above)
Comment (danielpeintner) [reply data/input_2022/TD/node-wot/node-wot.csv]: I see. I just just picked another sample that I think came from you ;-)
https://github.com/w3c/wot-testing/blob/main/data/input_2022/TD/editdor/editdor.csv

**Pull request merged**